Github feature request template: Fix label & improve paragraph order (#2469)
jamulussoftware/jamulus: .github/ISSUE_TEMPLATE/feature_request.md

name: Specified Feature
about: Agreed specification for a feature or enhancement
title: ''
labels: 'feature request'
assignees: ''
---

**Provide context**

> <!-- What's the status quo and how is this non-optimal?** -->

Reviewer: I don't think this is clear enough (at least for me "giving context" could mean anything). Can you formulate the headline as question too?

Maybe: What is the current behaviour and why should it be changed?
Comment: Please give a precise description of the status quo and how this is not optimal.

Author: Updated (more or less verbatim).

> **Describe possible approaches**

Reviewer: I think the user should also be lead to think about KISS. Can this be accieved by another way? Another software? Why should this be supported by Jamulus?
Comment: Please note the contribution guidelines (link here), especially that new features should be added with caution to keep Jamulus simple.

Author: Updated. Can you re-check?

I agree with all of your points, but I think it has downsides to have rather long explanations, too. People may either ignore it or walk away with some "too complicated" feeling.

But as I mainly care about the order, I've added your suggestions.

Reviewer: Yes, thought of that too. Let me think about something else...

<!--

> If multiple approaches are possible, list them here with pros and cons.

Reviewer: ```suggestion
Think about other approaches and list them here with pros and cons if possible.
```

Reference any relevant discussion threads if possible, and consider tagging the main participants.
-->

<!-- Describe the solution you'd like -->

**Has this feature been discussed and generally agreed?**

> <!-- Don't worry about requesting something that hasn't been discussed - we'll just move it to Discussion if we think it's unclear. But ideally, issues should have an agreed specification such that anyone could action it without much help. -->

Reviewer: ```suggestion
<!-- Don't worry about requesting something that hasn't been discussed - we'll just move it to Discussion if we think it's unclear. But ideally, issues should have an agreed, clear specification such that anyone could action it without much help. -->
```

name: Specified Feature
about: Agreed specification for a feature or enhancement
title: ''
labels: 'feature request'
assignees: ''
---

**What is the current behaviour and why should it be changed?**
<!-- Please give a precise description of the status quo and why this is not optimal. -->

**Describe possible approaches**
<!--

> List possible approaches here with pros and cons if possible.
> Consider both Jamulus-internal changes as well as external interfaces or solutions.
> Reference any relevant discussion threads if possible, and consider tagging the main participants.
> Please note the contribution guidelines (https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md), especially that new features should be added with caution to keep Jamulus simple.

Reviewer: ```suggestion
List possible approaches with pros and cons, reference any relevant discussion threads if possible, and consider tagging the main participants.
Please note the contribution guidelines (https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md), especially the part about "keeping it simple and stupid".
```
